test/e2e/common/node/pod_resize.go

```diff
@@ -894,19 +931,19 @@ func doPodResizeTests(f *framework.Framework) {
 			containers: []e2epod.ResizableContainerInfo{
```

Comment:
Can we parallelize this test case that modifies nodes for an extended resource? Are there any side effects to other tests?

Reply:
Hmm. Good catch. So far I do not see any side-effects because we have just one test for extended resource. I expect this to cause an issue in future though.